doc/design/image-management.md

```diff
@@ -0,0 +1,26 @@
+### How VIC manages filesystem images
+
+A big part of a container runtime is how it uses and manages filesystem images. There are various approaches to managing container images, but the net result needs to be enough binaries to run the main container process. Those binaries may infact be a single statically-linked binary, or it could be a operating system dependencies, third party libraries etc.
+
+Given that each containerVM in VIC does not share a filesystem with anything else, for a container VM to be functional, it requires the following:
+* OS binaries from which to boot. These are contained in the ISO attached to the containerVM
+* A disk, representing the container filesystem, mounted at "/". The disk may contain OS libraries, but must contain the binaries corresponding to the entry point of the container. The disk will typically be R/W and the underlying implementation may capture all reads into a layer which can be committed into a snapshot
+* 0 or more R/W disks which persist beyond the lifespan of the container
+
+#### Port Layer Storage
+
+#### Imagec
```

Comment:
Add a link to/from doc/design/components.md#imagec and consider whether that description needs updating.